src/ssl/ProxyCerts.h

```diff
@@ -11,22 +11,51 @@
 
 #if USE_OPENSSL
 #include "acl/forward.h"
+#include "acl/Gadgets.h"
+#include "ssl/gadgets.h"
 
 class sslproxy_cert_sign
 {
 public:
-    int alg;
-    ACLList *aclList;
-    sslproxy_cert_sign *next;
+    sslproxy_cert_sign() = default;
+    sslproxy_cert_sign(const sslproxy_cert_sign &) = delete; // prohibit all copy/move
+    ~sslproxy_cert_sign() {
+        if (aclList)
+            aclDestroyAclList(&aclList);
```

Comment:
This non-trivial destructor that needs external code for its implementation should be moved to a `.cc` file IMO, but I do not insist on this change. Your call.

Reply:
This one I will leave in the .h since there are only one place each is used.

Reply:
> since there are only one place each is used

FWIW, the number of current callers is a poor (primary) justification for creating inlining mistakes because it increases technical debt: Even if incorrect inlining does not create immediate problems, the developer adding another caller is unlikely to notice the problem they triggered, and even if they do notice, fixing that problem would be out of their work scope.